src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser.SymbolSpec.cs

```diff
@@ -0,0 +1,245 @@
+                        break;
+                    case "struct":
+                        builder.Add(_struct);
+                        break;
+                    case "interface":
+                        builder.Add(_interface);
+                        break;
+                    case "enum":
+                        builder.Add(_enum);
+                        break;
+                    case "property":
+                        builder.Add(_property);
+                        break;
+                    case "method":
+                        builder.Add(_method);
+                        break;
+                    case "field":
+                        builder.Add(_field);
+                        break;
+                    case "event":
+                        builder.Add(_event);
+                        break;
+                    case "namespace":
+                        builder.Add(_namespace);
+                        break;
+                    case "delegate":
+                        builder.Add(_delegate);
+                        break;
+                    case "type_parameter":
+                        builder.Add(_typeParameter);
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            return builder.ToImmutableAndFree();
+        }
+
+        private static ImmutableArray<Accessibility> GetSymbolsApplicableAccessibilities(
+            string symbolSpecName,
+            IReadOnlyDictionary<string, object> conventionsDictionary)
+        {
+            if (conventionsDictionary.TryGetValue($"dotnet_naming_symbols.{symbolSpecName}.applicable_accessibilities", out object result))
+            {
+                return ParseAccessibilityKindList(result as string ?? string.Empty);
+            }
+
+            return ImmutableArray<Accessibility>.Empty;
+        }
+
+        private static ImmutableArray<Accessibility> ParseAccessibilityKindList(string symbolSpecApplicableAccessibilities)
+        {
+            if (symbolSpecApplicableAccessibilities == null)
+            {
+                return ImmutableArray<Accessibility>.Empty;
+            }
+
+            var builder = ArrayBuilder<Accessibility>.GetInstance();
+            if (symbolSpecApplicableAccessibilities.Trim() == "*")
+            {
+                builder.AddRange(Accessibility.Public, Accessibility.Internal, Accessibility.Private, Accessibility.Protected, Accessibility.ProtectedOrInternal);
+                return builder.ToImmutableAndFree();
+            }
+
+            foreach (var symbolSpecApplicableAccessibility in symbolSpecApplicableAccessibilities.Split(',').Select(x => x.Trim()))
+            {
+                switch (symbolSpecApplicableAccessibility)
+                {
+                    case "public":
+                        builder.Add(Accessibility.Public);
+                        break;
+                    case "internal":
```

Comment:
As discussed offline, maybe we can just use either language's nomenclature interchangeably.